atlasdb-impl-shared/build.gradle

```diff
@@ -33,6 +33,7 @@ dependencies {
   processor 'com.google.auto.service:auto-service:1.0-rc2'
   processor project(":atlasdb-processors")
 
+  testCompile project(":atlasdb-tests-shared")
```

Comment:
This okay? Was needed to use the AbstractTransactionTest constants everywhere.

Reply:
Hmm, seems wrong that "impl" should depend on "tests". "tests" already depends on "impl"!

Reply:
Either move to a new utility file on impl-shared (`TransactionTestConstants`?) or keep them in tests-shared, and duplicate where necessary on impl-shared.